src/periodograms.jl

```diff
@@ -6,6 +6,7 @@ using ..Util, ..Windows
 export arraysplit, nextfastfft, periodogram, welch_pgram, mt_pgram,
        spectrogram, power, freq, stft
 using FFTW
+import FFTW: Frequencies, fftfreq, rfftfreq
```

Comment:
```suggestion
```
I believe this isn't necessary, since there's a `using FFTW` and FFTW reexports the symbols from AbstractFFTs. Same with the import from DSP in the tests.